.github/workflows/main.yml

```diff
@@ -18,4 +18,5 @@ jobs:
       - name: Start MongoDB Server
         uses: ./
         with:
+          mongodb-image: 'public.ecr.aws/docker/library/mongo'
```

Comment:
I would like to avoid adding the `mongodb-image` configuration to all GitHub Action workflows. It would be good if we add a new workflow file using the ECR registry.

@Sam-Bate-ITV Sam, can you please remove all the added `mongodb-image` configs from the existing workflow YAML files and add a workflow. The new workflow file should be a copy of the `test-single-instance.yml` that uses a custom image.

Reply:
I've reverted this to use the default from docker hub. Of course this may impact the workflow from completing